packages/cactus-plugin-persistence-fabric/README.md

```diff
@@ -0,0 +1,225 @@
+# `@hyperledger/cactus-plugin-persistence-fabric`
+
+This plugin allows `Cactus` to persist Fabric Block general data and basic information about transactions into some storage (currently to a `PostgreSQL` database, but this concept can be extended further).
+Data in the database can later be analyzed and viewed in a GUI tool.
+GUI tool is in project root directory of Cacti project in GUI folder.
```

Comment:
Can you link to the folder containing the GUI tool, just for precision?

Reply:
> Can you link to the folder containing the GUI tool, just for precision?

done